packages/plugins/test/mocha/cli/propose.spec.ts

```diff
@@ -155,24 +155,28 @@ describe('Propose CLI command', () => {
               functionName: 'deploy',
               referenceName: 'GnosisSafe',
               variables: {},
+              value: '0',
```

Comment:
I didn't add any tests for airdropping or transferring value to the proposal test suite because I don't think that would cover anything not already covered by the current deploy command tests and unit tests. 

However, I am adding some tests to the website since there is some separate logic there that needs to be covered.